webrtc.html

```diff
@@ -9158,11 +9158,16 @@ <h3><dfn>RTCIceTransportState</dfn> Enum</h3>
               gathering, received an indication that there are no more remote
               candidates, finished checking all candidate pairs, and all pairs
               have either failed connectivity checks or have lost consent.
-              This is a terminal state.  Since an ICE restart may cause
-              connectivity to resume, entering the <code>failed</code> state
-              does not cause DTLS transports, SCTP associations or the data
-              channels that run over them to close, or tracks to mute.
-              </td>
+              This is a terminal state unless ICE is restarted. Since an ICE
+              restart may cause connectivity to resume, entering the
+              <code>failed</code> state does not cause DTLS transports, SCTP
+              associations or the data channels that run over them to close, or
+              tracks to mute.</td>
+              <div class="note">
+                An ICE restart can happen due to the <code>restartIce()</code>
+                method or the remote endpoint triggering renegotiation with new
+                ICE credentials.
+              </div>
```

Comment:
this needs to be moved inside the `<td></td>`

Reply:
Ack!

Reply:
Will remove note per jan-ivar's comment